content/en/user-guide/tools/event-studio/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Event Studio"
+weight: 12
+description: >
+  Event Studio allows you to debug, profile, and test events locally while developing event-driven applications!
```

Comment:
would  remove event-driven applications since any inter aws service communication is an event in our case

Event Studio allows you to record, trace, debug and replay any events exchanged between AWS services.